src/test/cypress/e2e/login.cy.ts

```diff
@@ -34,4 +34,53 @@ describe("Login Flow", () => {
     // user doesn't exist or password is incorrect
     cy.contains("p", "Invalid username or password.").should("be.visible");
   });
+
+  it("REALM_SSO_SESSION_MAX_IN_FLIGHT_LOGINS_PER_USER configuration test", () => {
+    cy.getValueFromSecret("keycloak", "keycloak-realm-env", "REALM_SSO_SESSION_MAX_IN_FLIGHT_LOGINS_PER_USER").then(decoded => {
+      const secretLimit = parseInt((decoded || "").trim(), 10);
+      expect(Number.isFinite(secretLimit), `Decoded secret value should be a number, got: '${decoded}'`).to.be.true;
```

Comment:
To be honest the whole test feels a bit trivial and like it might fit better as a helm test on the uds-core side? Essentially it's just checking that the arg is set on the pods + matches the secret value I think.

Reply:
Great point! I'll do that!